src/OmniSharp/Models/RunCodeActionResponse.cs

```diff
@@ -5,6 +5,6 @@ namespace OmniSharp.Models
     public class RunCodeActionResponse
     {
         public string Text { get; set; }
-        public IEnumerable<LinePositionSpanTextChange> Changes { get; set; }
+        public IEnumerable<ModifiedFileResponse> Changes { get; set; }
```

Comment:
opting for a /v2/runcodeaction route where we make this change and also use string identfiers instead of index-based identifiers